### PR TITLE
Add com.ashwingohil.petalssticky

### PR DESCRIPTION
**Pull request by ashwingohil**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

File: com.ashwingohil.petalssticky.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+    "app-id" : "com.ashwingohil.petalssticky",
```
Comment (hfiguiere) [inline com.ashwingohil.petalssticky.json:2]: `ashwingohil.com` doesn't exist.

File: com.ashwingohil.petalssticky.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    "modules" : [
+        "pypi-dependencies.json",
+        {
+            "name" : "petalssticky",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/ashwingohil/petalssticky"
```
Comment (hfiguiere) [inline com.ashwingohil.petalssticky.json:34]: this is 404.

Also a commit/tag is required.

File: com.ashwingohil.petalssticky.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
+{
+    "app-id" : "com.ashwingohil.petalssticky",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "44",
```
Comment (bbhtt) [inline com.ashwingohil.petalssticky.json:4]:
```suggestion
    "runtime-version" : "45",
```
It is the latest runtime

File: pypi-dependencies.json [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+            "sources": [
+                {
+                    "type": "file",
+                    "url": "https://files.pythonhosted.org/packages/fc/b8/ff33610932e0ee81ae7f1269c890f697d56ff74b9f5b2ee5d9b7fa2c5355/python_xlib-0.33-py2.py3-none-any.whl",
+                    "sha256": "c3534038d42e0df2f1392a1b30a15a4ff5fdc2b86cfa94f072bf11b10a164398"
+                }
+            ]
+        },
+        {
+            "name": "six",
```
Comment (bbhtt) [inline pypi-dependencies.json:54]: Six in the runtime drop this https://gitlab.com/freedesktop-sdk/freedesktop-sdk/-/blob/44cef1fe38f6ae2be7d66d63ee5abf64b2d3e67c/elements/platform.bst#L73

**Comment by hfiguiere** [comment]
To build the Flatpak on Flathub, the "source" must be publicly accessible. And by "source" this can be build artifacts if you are denying the source code freedom, or if your build system is broken.

**Comment by ashwingohil** [comment]
Hi,

Yes, to my knowledge, I know 45 will be supported for longer. *Reason for
using runtime 44* is that somehow when I use libadwaita 4..1.0 which is
available in 45, my app gets broken. It works fine with available Gtk4 and
Libadwaita in runtime 44. [gtk - 4.10.5 and libadwaita-1 - 1.3.5]

I understand now that private repos will not clone. So I will make the
necessary changes to manifest and host my source public. (in 2-3 days)

Thanks,
A

**Pull request closed without merging**